### ISSUE TITLE
es版本和ik分词器的版本问题

### ISSUE THREAD

**Issue by NoviceEric** [medcl/elasticsearch-analysis-ik#909]
我集群里面安装的是es 6.8.19版本，但是在这里只能找到 6.8.18版本的 ik分词器
强行安装6.8.18 ik导致es服务无法启动
是否一定要安装对应版本的ik呢

**Comment by qaz40520** [comment]
yes, my solution is edit `pom.xml` setting version for es version, then use mvn compile again

```
mvn clean
mvn compile
mvn package
```

**Comment by 52Hertz-C** [comment]
下载源码，把版本改掉，重新编译，elasticsearch-analysis-ik-x.x.x/target/releases/elasticsearch-analysis-ik-x.x.x.zip，解压就是插件。
![image](https://user-images.githubusercontent.com/39614241/142988884-b60764ba-743c-44cf-a49b-f03d769226d2.png)
